CHANGELOG.md

```diff
@@ -0,0 +1,140 @@
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-160'>AUTH-160</a>] -         Release Notes
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-162'>AUTH-162</a>] -         Automate regression testing for LDAP Authentication
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-184'>AUTH-184</a>] -         Create a list of 3rd Party Software Dependencies
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-193'>AUTH-193</a>] -         Stable Infrastructure Chart (with Identity Service)
+</li>
+</ul>
+
+<h2>Bug
+</h2>
+<ul>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-120'>AUTH-120</a>] -         Pass the secret name as values to identity service 0.2 helm chart
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-144'>AUTH-144</a>] -         alfresco-anaxes-hello-world documentation steps inconsistency
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-145'>AUTH-145</a>] -         alfresco-anaxes-hello-world-ui throws SyntaxError in browser console
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-152'>AUTH-152</a>] -         ACS6: New identity-service authentication sub-system can freeze server
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-171'>AUTH-171</a>] -         Token request with SAML provider results with &quot;upstream sent too big header&quot;
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-172'>AUTH-172</a>] -         Identity Service Ingress is Broken Due to Truncation Differences
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-178'>AUTH-178</a>] -         Empty Realm in Identity Service
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-181'>AUTH-181</a>] -         Enable per-namespace host names in Bamboo-driven SAML tests
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-182'>AUTH-182</a>] -         Install jq binary in Bamboo elastic agents
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-183'>AUTH-183</a>] -         Keycloak stateful sets are left over after &quot;helm delete&quot;
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-189'>AUTH-189</a>] -         Keycloak PostgreSQL startup error when using helm upgrade
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-198'>AUTH-198</a>] -         Error message overflows login box
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-207'>AUTH-207</a>] -         APS http redirect_uri parameter for https-only requests
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-211'>AUTH-211</a>] -         Error pages don&#39;t render correctly
+</li>
+</ul>
+
+<h2>        Task
+</h2>
+<ul>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-15'>AUTH-15</a>] -         Move the activity-sso-idm project from the Activiti org to the Alfresco org
```

Comment:
This would probably be better placed in Activiti release notes.

Reply:
Good point.  I removed AUTH-15.